### PR TITLE
Update the component doc template

### PR DESCRIPTION
**Pull request by JessamyT**

### CODE REVIEW

File: docs/components/component/_index.md [hunk 1]
```diff
@@ -27,45 +27,124 @@ Most robots with a COMPONENT need at least the following hardware:
...
+[`model1`](model1/) | An XYZ unit made by Some Company(INSERT LINK AS APPLICABLE)
+[`model2`](model2/) | An ABCDE unit made by Some Other Company(INSERT LINK AS APPLICABLE)
+
+If you want to use another COMPONENT model with Viam, you can [define a custom component](../../program/extend/).
+
+## API
+
+The COMPONENT component supports the following methods:
+
+*Use the method names in the protobuf, not the Python or Go-specific method names.*
```
Comment (npentrel) [inline docs/components/component/_index.md:43]:
```suggestion
*Writing Instructions: Use the method names in the protobuf, not the Python or Go-specific method names.*
```
Just to be clear haha

File: docs/components/component/model1.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+{{< tabs >}}
+{{% tab name="Config Builder" %}}
+
+On the **COMPONENTS** sub-tab, navigate to the **Create Component** menu.
+Enter a name for your arm, select the `arm` type, and select the `model1` model.
+
+![Creation of a `model1` arm in the Viam app config builder.](../img/model1-builder.png)
+
+{{% /tab %}}
```
Comment (npentrel) [inline docs/components/component/model1.md:23]:
```suggestion
Click **Create Component** and then fill in the attributes for your model.

{{% /tab %}}
```

**Review by npentrel — APPROVED** [review]
Two comments otherwise looks good!

File: docs/components/component/model1.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+# SMEs:
+---
+
+The `model1` arm model supports the XYZ unit made by Some Company(INSERT LINK AS APPLICABLE).
+Optional additional description/information.
+
+{{< tabs >}}
+{{% tab name="Config Builder" %}}
+
+On the **COMPONENTS** sub-tab, navigate to the **Create Component** menu.
```
[the remark this answers is not in the document]
Comment (JessamyT) [reply docs/components/component/model1.md]: good call

File: docs/components/component/model1.md [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+---
+
+The `model1` arm model supports the XYZ unit made by Some Company(INSERT LINK AS APPLICABLE).
+Optional additional description/information.
+
+{{< tabs >}}
+{{% tab name="Config Builder" %}}
+
+Navigate to the **config** tab of your robot's page in [the Viam app](https://app.viam.com).
+Click on the **Components** sub-tab and navigate to the **Create component** menu.
```
Comment (JessamyT) [inline docs/components/component/model1.md:19]:
```suggestion
Click on the **Components** subtab and navigate to the **Create component** menu.

```

File: docs/components/component/model1.md [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+Enter a name for your arm, select the `arm` type, and select the `model1` model.
+
+![Creation of a `model1` arm in the Viam app config builder.](../img/model1-builder.png)
+
+Click **Create Component** and then fill in the attributes for your model.
```
Comment (JessamyT) [inline docs/components/component/model1.md:24]:
```suggestion
Click **Create component** and then fill in the attributes for your model.
```

File: docs/components/component/model1.md (hunk 2 above)
Comment (JessamyT) [reply docs/components/component/model1.md:27]: done (with lowercase c)

File: docs/components/component/_index.md [hunk 6]
```diff
@@ -27,45 +27,124 @@ Most robots with a COMPONENT need at least the following hardware:
 
 ## Configuration
 
-Configuring this component on your robot enables you to ....
-You can easily configure your component, as pictured below, on the [Viam app](https://app.viam.com/).
-{{< tabs name="TabPanelExample" >}}
-{{% tab name="UI"%}}
+For configuration information, click on your COMPONENT's model:
```
Comment (sguequierre) [inline docs/components/component/_index.md:30]: I like this but I also think that `supported models include` is worthwhile, do you have any ideas for an in-between/combo that wouldn't make it look extra wordy? I don't really 😆 other than that LGTM

**Review by sguequierre — APPROVED** [review]
LGTM One comment

File: docs/components/component/_index.md (hunk 6 above)
Comment (JessamyT) [reply docs/components/component/_index.md:30]: Yeah I agree. Hmm. Maybe this?
```suggestion
Find configuration information for each supported model at the links below:
```

Comment (npentrel) [reply docs/components/component/_index.md:30]: How about: 
```suggestion
The COMPONENT component supports the following models. Click on the component model name for detailed configuration information. 
```

Comment (JessamyT) [reply docs/components/component/_index.md:30]:
```suggestion
The COMPONENT component supports the following models.
Click on the component model name for detailed configuration information.
```
I like that! Just adding a newline and removing a trailing space. Do we throw a colon on the end or no?

Comment (npentrel) [reply docs/components/component/_index.md:30]: Either works so go for whichever you prefer :)

**Pull request merged**